dissect/target/plugins/os/unix/config.py

```diff
@@ -61,11 +74,19 @@ def _collapse_dict(self, dictionary: dict, collapse: bool = False) -> dict[str,
         return new_dictionary
 
 
-class Ini(LinuxConfigurationParser):
+class Ini(ConfigurationParser):
     def __init__(self, collapse: Optional[Union[bool, set]] = True) -> None:
```

Comment:
This is missing `separator` and `comment_prefixes` defaults, which will lead to issues with `ParserConfig.create_parser()`